examples/meta/src/regression/chaidtree.sg.in

```diff
@@ -0,0 +1,29 @@
+File f_feats_train    = read_csv("@SHOGUN_DATA@/regression_1d_linear_features_train.dat")
+File f_feats_test     = read_csv("@SHOGUN_DATA@/regression_1d_linear_features_test.dat")
+File f_labels_train   = read_csv("@SHOGUN_DATA@/regression_1d_linear_labels_train.dat")
+
+#![create_features]
+Features feats_train = create_features(f_feats_train)
+Features feats_test = create_features(f_feats_test)
+Labels labels_train = create_labels(f_labels_train)
+#![create_features]
+
+#![set_feature_types]
+IntVector ft(1)
+ft[0] = 2
+#![set_feature_types]
+
+#![create_machine]
+Machine chaidtree = create_machine("CHAIDTree", dependent_vartype=2, feature_types=ft, num_breakpoints=50)
+chaidtree.set_labels(labels_train)
```

Comment:
Could you pass labels either to the constructor or use put instead please?

Reply:
Okay, I'll do that.